packages/core/src/ChatMessage/ChatMessage.tsx

```diff
@@ -35,6 +35,13 @@ const Message = styled(Flex)`
   ${applyVariations('ChatMessage', variations)};
 `
 
+const IconWrapper = styled(Flex)`
+  background-color: ${getPaletteColor('primary.base')};
+  border-radius: 16px;
```

Comment:
n-b radius, w, and h should be jsx props

Reply:
Updated and removed the styled component completely